### PR TITLE
Exotic weapons

### PR DESCRIPTION
**Pull request by Mythacri**
New exotic weapon types and seven new weapon properties have been added.

### CODE REVIEW

File: scripts/modules/system-config.mjs [hunk 1]
```diff
@@ -94,18 +94,27 @@ export class SystemConfig {
...
     });
   }
 
   static _weaponProficiencies() {
     // The sections in the weapon proficiency config.
     foundry.utils.mergeObject(CONFIG.DND5E.weaponProficiencies, {
       firearmRen: "MYTHACRI.WeaponProficiencyFirearmRenPl",
-      firearmInd: "MYTHACRI.WeaponProficiencyFirearmIndPl"
+      firearmInd: "MYTHACRI.WeaponProficiencyFirearmIndPl",
+      Exotic: "MYTHACRI.WeaponProficiencyExoticPl"
```
Comment (krbz999) [inline scripts/modules/system-config.mjs:117]:
```suggestion
      exotic: "MYTHACRI.WeaponProficiencyExoticPl"
```

File: scripts/modules/system-config.mjs [hunk 2]
```diff
@@ -118,7 +127,8 @@ export class SystemConfig {
     // The weapon type itself, as an option in an item sheet.
     foundry.utils.mergeObject(CONFIG.DND5E.weaponTypes, {
       firearmRen: "MYTHACRI.WeaponProficiencyFirearmRen",
-      firearmInd: "MYTHACRI.WeaponProficiencyFirearmInd"
+      firearmInd: "MYTHACRI.WeaponProficiencyFirearmInd",
+      Exotic: "MYTHACRI.WeaponProficiencyExotic"
```
Comment (krbz999) [inline scripts/modules/system-config.mjs:131]:
```suggestion
      exotic: "MYTHACRI.WeaponProficiencyExotic"
```

File: scripts/modules/system-config.mjs [hunk 3]
```diff
@@ -141,6 +151,14 @@ export class SystemConfig {
 
     CONFIG.DND5E.weaponIds.fishhook = "mythacri-shared-compendium.equipment-myth.lQqGpkcldeOKwZM0";
     CONFIG.DND5E.weaponIds.shovel = "mythacri-shared-compendium.equipment-myth.1D0WRHGVgbjaikkM";
+    CONFIG.DND5E.weaponIds.machete = "mythacri-shared-compendium.equipment-myth.06hcru0PpE0Q33Av";
+
+    CONFIG.DND5E.weaponIds.volleyGun = "mythacri-shared-compendium.equipment-myth.HHePiOzWRtjYjcl6";
+    CONFIG.DND5E.weaponIds.duckfootPistol = "mythacri-shared-compendium.equipment-myth.GthTZpxLEVrMHQMm";
+    CONFIG.DND5E.weaponIds.Greatspear = "mythacri-shared-compendium.equipment-myth.YK1KjJ1qtDJlppVd";
```
Comment (krbz999) [inline scripts/modules/system-config.mjs:158]:
```suggestion
    CONFIG.DND5E.weaponIds.greatspear = "mythacri-shared-compendium.equipment-myth.YK1KjJ1qtDJlppVd";
```

File: scripts/modules/system-config.mjs [hunk 4]
```diff
@@ -94,18 +94,27 @@ export class SystemConfig {
       parrying: "MYTHACRI.WeaponPropertyParry",
       scatter: "MYTHACRI.WeaponPropertyScatter",
       superheavy: "MYTHACRI.WeaponPropertySuperheavy",
+      concealable: "MYTHACRI.WeaponPropertyConcealable",
+      parrying: "MYTHACRI.WeaponPropertyParrying",
```
[the remark this answers is not in the document]
Comment (Mythacri) [reply scripts/modules/system-config.mjs]: Second parrying property has been removed.

**Review by krbz999 — APPROVED** [review]
LGTM!

**Pull request merged**